cmd/oras/internal/option/target.go

```diff
@@ -118,24 +116,30 @@ func (opts *Target) Parse() error {
 }
 
 // parseOCILayoutReference parses the raw in format of <path>[:<tag>|@<digest>]
-func parseOCILayoutReference(raw string) (path string, ref string, err error) {
+func (opts *Target) parseOCILayoutReference() error {
+	raw := opts.RawReference
+	var path string
+	var ref string
+	var err error
```

Comment:
This line should be in between of line 130 and line 131 since it is only used in the `else` block.

Reply:
moved